core/src/Cabana_Experimental_NeighborList.hpp

```diff
@@ -152,10 +152,9 @@ struct NeighborDiscriminatorCallback2D_FirstPass
 {
     Counts counts;
     using tag = ArborX::Details::InlineCallbackTag;
-    template <typename Predicate, typename OutputFunctor>
+    template <typename Predicate>
     KOKKOS_FUNCTION void operator()( Predicate const &predicate,
-                                     int primitive_index,
-                                     OutputFunctor const & ) const
+                                     int primitive_index ) const
```

Comment:
I am taking advantage of the new `BVH::query()` overload recently added (within last month), which require updating ArborX.  I can revert if you think that is an issue.

Reply:
I have no issue with that